ghostscript/ghostscript-system-zlib.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/base/lib.mak b/base/lib.mak
```

Comment:
Same as above, can we add a comment at the top please, of where the original patch comes from?

Reply:
of course 🙉